lib/brakeman/processors/controller_processor.rb

```diff
@@ -192,8 +192,8 @@ def add_fake_filter exp
 
     filter_name = ("fake_filter" + rand.to_s[/\d+$/]).to_sym
     args = exp.block_call.arglist
-    args.insert(1, Sexp.new(:lit, filter_name))
-    before_filter_call = make_call(nil, :before_filter, args)
+    args.insert(1, Sexp.new(:lit, filter_name).line(exp.line))
+    before_filter_call = make_call(nil, :before_filter, args).line(exp.line)
```

Comment:
These failed on my non-nil change?

Reply:
This may have been general cleanup to ensure Sexps have line numbers to prevent problems later.